sources/OnOffStrategy/ConfigurationRepository.php

```diff
@@ -7,4 +7,11 @@ interface ConfigurationRepository
     public function isEnabled(string $feature): bool;
 
     public function setEnabled(bool $enabled, string $feature): void;
+
+    /**
+     * @return string[]
+     */
+    public function listFeatures(): array;
+
+    public function removeFeature(string $feature): void;
```

Comment:
Since these methods are duplicated in all `ConfigurationRepository` interfaces, I think we should introduce a new shared interface that is not about "configuration" but more about "administration" (or a better name if you find one).